WeatherMonitoringAndReportingService.BL/Utilities/ConsoleOutputUtility.cs

```diff
@@ -0,0 +1,30 @@
+using WeatherMonitoringAndReportingService.BL.Enums;
+
+namespace WeatherMonitoringAndReportingService.BL.Utilities;
+
+public static class ConsoleOutputUtility
+{
+    public static DataFormat? GetDataFormat()
+    {
+        Console.WriteLine("Hello there!");
+        Console.WriteLine("Welcome to our weather monitoring and reporting service!");
+        Console.WriteLine("What data format do you like to enter? 1 for JSON format, and 2 for XML format!");
+
+        var dataFormatChoice = Console.ReadLine();
+
+        while (dataFormatChoice != "1" && dataFormatChoice != "2")
+        {
+            Console.WriteLine("Your number should be eiter 1 or 2 only.");
+            dataFormatChoice = Console.ReadLine();
+        }
+        
+        var dataFormatChoiceInt = int.Parse(dataFormatChoice);
```

Comment:
maybe you can simplify this part by using int.trypars(... , int out dataFormatChoiceInt)

Reply:
That would be too generic for my use case, I wanna make it valid only if the user entered either 1 or 2.